rumydata/table.py

```diff
@@ -432,7 +432,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     @staticmethod
     def _row_handler(row: list) -> List[str]:
-        return [str(x or '') for x in row]
+        return [str(x) or '' for x in row]
```

Comment:
```suggestion
        return ['' if x is None else str(x) for x in row]
```

Reply:
Doing it this way explicitly checks for the `None`, which is what this handler was primarily looking for in the first place. If we just coerce a None value via `str(None)`, that will cause the row handler to pass empty cells with a value of None instead of keeping them empty.

Reply:
I'm realizing now that this `_row_handler` method is specific to the `ExcelFile` class, and getting at what you were saying about needing to run tests through each type. That seems like it could be never ending, but I think we can be fairly target about it. The **biggest** difference that can be introduced by different types is the way that this row handler operates; everything else gets handled by the same code under the row level. So in terms of writing tests, what I think we need to do is write some specific row tests which do this:

1. Create a table which tries to test for the row-level edge cases
2. Write that table into an Excel file (and for that matter, Parquet as well)
3. Test that table for the expected errors